cypress/support/e2e.ts

```
@@ -0,0 +1,13 @@
+// Import commands.js using ES2015 syntax:
+import './commands'
```

ESLint

[error] './commands' import is restricted from being used by a pattern.

(no-restricted-imports)